client/package.json

```diff
@@ -76,14 +76,25 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "0.901.9",
+    "@angular-eslint/builder": "1.2.0",
+    "@angular-eslint/eslint-plugin": "1.2.0",
+    "@angular-eslint/eslint-plugin-template": "1.2.0",
+    "@angular-eslint/schematics": "1.2.0",
+    "@angular-eslint/template-parser": "1.2.0",
     "@angular/cli": "9.1.9",
     "@angular/compiler-cli": "9.1.11",
     "@angular/language-service": "9.1.11",
     "@fortawesome/fontawesome-free": "5.13.1",
     "@types/jasmine": "3.5.11",
     "@types/jasminewd2": "2.0.8",
     "@types/node": "^12.12.30",
+    "@typescript-eslint/eslint-plugin": "4.3.0",
+    "@typescript-eslint/parser": "4.3.0",
     "codelyzer": "5.2.1",
+    "eslint": "^7.6.0",
```

Comment:
We use specific versions since we're not a library, so this caret can be removed in favor for the latest stable 7.6.X version number here that satisfies the other eslint dependencies added.